examples/hybrid-compute/README.md

```diff
@@ -0,0 +1,85 @@
+
+## Organization
+
+This subfolder is organized as follows.
+
+---
+    ├── Hybrid-compute        <- Examples of how to use Hybrid Compute via WASM
+          |
+          ├── RUS_WASM        <- Repeat Until Success source code
+---
+
+## Download
+
+If you haven't done so already, download the complete set of examples on the user portal by clicking the **Download** button on the bottom-left of the folder viewer.
+
+## Notebook Example
+
+The python notebook in this directory called `repeat_until_success_with_wasm.ipynb` can be run to show an example of the hybrid compute feature's workflow, without needing to edit any classical functions or compile to Wasm.
+
+## Setup
+
+To create and use your own Hybrid Compute functions in your workflow, it is necessary to use Wasm and a language that compiles code to Wasm. These instructions contain the steps for setting up an environment to do this.
+
+It is recommended to either use Rust or C++ for the compilation language to Wasm. The languages have functionality that easily compiles to Wasm. For other language options see [Wasm Getting Started Developer's page](https://webassembly.org/getting-started/developers-guide/).
+
+### Option 1: Rust
+
+If you're new to Rust, see the [The Rust Programming Language](https://doc.rust-lang.org/book/title-page.html) book. Note that Cargo is the Rust package manager. For more info on Cargo, see [The Cargo Book](https://doc.rust-lang.org/cargo/guide/).
+
+If using Visual Studio code, you can also install the `rust-analyzer` extension in the Extensions tab.
+
+#### Install Rust
+
+Install Rust using the [Rust Installer](https://www.rust-lang.org/tools/install).
+
+**Windows Users:** Before running the installer, you'll need to install the [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
+
+#### Create a new Rust project
+
+Navigate to wherever you plan to store your Rust project files, and run: `cargo new --lib rus_wasm`. The project name `rus_wasm` can be whatever project name you want to use.
+
+This creates a new library in a subdirectory with the project name you used with everything you need to get going.
+
+* The `/src/lib.rs` file contains some generated Rust code. This code can be erased and replaced with code you plan to use in your hybrid compute program.
+* Inside this subdirectory is a file called `Cargo.toml`. This file will be used to configure your build.
+
+#### Define classical functions in Rust
+
+Define the classical functions you plan to use in your hybrid compute program in the `src/lib.rs` file. Examples of the types of functions that can be put in Rust code are provided in the `Rust` folders within each of the example folders. *We recommend looking at one of the provided examples to know which functions and attributes are required for Hybrid Compute to work.*
```

Comment:
Do you maybe want to give more details here about the decorators and more detailed declaration?
(Maybe give a two line example here in the readme?
```
#[no_mangle]
pub extern "C" fn wasm_func(i32) -> (i32) {

Reply:
At some point we need to add that all external libs like rand, print, ... are not supported and will cause the wasm calls to fail, even if this is only used in functions that are not called from the circuit.

(I have seen this is written below as well, but maybe we should consider writing this at multiple point, people tend to miss this... )